ivy/array/experimental/sorting.py

```diff
@@ -42,3 +42,16 @@ def msort(
             )
         """
         return ivy.msort(self._data, out=out)
+
+
+# lexsort
+def lexsort(
+    self: ivy.Array,
+    /,
+    *,
+    axis: int = -1,
+    out: Optional[ivy.Array] = None,
+) -> ivy.Array:
+    """
+    """
+    return ivy.lexsort(self._data, axis=axis, out=out)
```

Comment:
What you did here is let the function call itself, as the tests shows that RecursionError: maximum recursion depth exceeded.

Reply:
And to be more precise, it is not that the problem is here, it is that this call targets another function that will call itself and form a loop.

Reply:
for instance, look at this chain of error: E   ivy.exceptions.IvyBackendException: tensorflow: lexsort: tensorflow: lexsort: tensorflow: lexsort: tensorflow: lexsort: tensorflow: lexsort: tensorflow: lexsort: tensorflow: lexsort: tensorflow: nested_map: tensorflow: nested_map: tensorflow: default: tensorflow: exists: maximum recursion depth exceeded

Reply:
it all begins after this step happened: ivy/functional/ivy/sorting.py:259: in lexsort
    return ivy.current_backend(keys).lexsort(

Reply:
Which then seems to call back to the frontend function

Reply:
I fixed the indentation error about the "self", it is not recognized because of this. I know still I have a recursion error also, I am working on it